lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -23,8 +23,9 @@ defmodule Mix.Tasks.Escript.Install do
       ~/.mix/escripts/foo
 
   For convenience, consider adding `~/.mix/escripts` directory to your
-  `PATH` environment variable. For more information, check the wikipedia
-  article on PATH: https://en.wikipedia.org/wiki/PATH_(variable)
+  `PATH` environment variable. For more information, check the [Wikipedia
+  article on PATH][PATH]
+  [PATH]: https://en.wikipedia.org/wiki/PATH_(variable)
```

Comment:
I think we're safe leaving this like it is now then, and possibly update when earmark fixes this bug :)